### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by bgutai**

### CODE REVIEW

**Review by rostyslav48 — APPROVED** [review]
Good job! Code is tidy and neat)👍

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_DAYS = 3;
+  const LONG_DAYS = 7;
+  const SHORT_DAYS_DISCOUNT = 20;
+  const LONG_DAYS_DISCOUNT = 50;
+  const RENTAL_PRICE = 40;
+  const TOTAL_PRICE = days * RENTAL_PRICE;
```
Comment (Moroz-Dmytro) [inline src/calculateRentalCost.js:36]: It is customary to name static variables with a capital letter, if something is calculated in the variable, it is better to name it using the usual camelCase.

**Review by Moroz-Dmytro — CHANGES_REQUESTED** [review]
Small comment
Also if you went through checklist set checkmark in the table
image

**Review by Anna-Tkachenko — APPROVED** [review]

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_DAYS = 3;
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:31]: Will be better to name it as `term` instead of `days`

**Pull request closed without merging**